purchases/src/main/kotlin/com/revenuecat/purchases/common/logUtils.kt

```diff
@@ -31,7 +31,7 @@ internal fun infoLog(message: String) {
 }
 
 // Public because it's been used in the hybrids
-fun warnLog(message: String) {
+internal fun warnLog(message: String) {
```

Comment:
I think we can make this one internal even if it's been used in the hybrids

Reply:
We're still using this in PHC, will it work after setting it to internal?

https://github.com/RevenueCat/purchases-hybrid-common/blob/db9cb9090cbe2959408c32d2ee122fc10f1af621/android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt#L381

Reply:
No... it's only two logs though... maybe we can do something so we can hide this in this SDK 🤔

Reply:
Well, if we make it internal, we can still log things in PHC/Hybrids... The consequence of that however is that those logs won't be redirected through the log handler in case the developer has set any. If we want to bypass that, we should wrap the log handler in PHC to set it in Android but also to redirect the messages in PHC. But honestly, if those logs are not very important, I would be ok if they are not redirected for now.

Reply:
Yes, I agree, I'd rather make this internal so we don't expose this to devs for no reason